src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,26 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String s = "";
```

Comment:
let's not forget about checklist before submitting your solution 
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html
double check your task, you have mistakes, mentioned there

Reply:
done